satellite/metainfo/metainfo.go

```diff
@@ -412,13 +412,6 @@ func (endpoint *Endpoint) DeleteSegmentOld(ctx context.Context, req *pb.SegmentD
 	}
 
 	if pointer.Type == pb.Pointer_REMOTE && pointer.Remote != nil {
-		for _, piece := range pointer.GetRemote().GetRemotePieces() {
-			_, err := endpoint.containment.Delete(ctx, piece.NodeId)
```

Comment:
Does metainfo still require containment?

Reply:
I don't think so. I will remove it.

Reply:
Done